argilla-server/src/argilla_server/models/database.py

```diff
@@ -480,16 +480,6 @@ class User(DatabaseModel):
         passive_deletes=True,
         order_by=Response.inserted_at.asc(),
     )
-    datasets: Mapped[List["Dataset"]] = relationship(
```

Comment:
Why deleting it?

Reply:
Ok, but it's fine to left the association there in case it's useful from the console, tests, etc.